accname/name/comp_text_node.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html>
+<head>
+  <title>Name Comp: Text Node</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<!-- I'm not certain whether #comp_text_node requires a lot of testing outside of the #comp_name_from_content contexts, -->
+<!-- but I did think of one example where text node versus comment node may make a difference when joining text nodes with a space vs innerText. -->
+
+<h2 class="ex" data-expectedlabel="heading label" data-testname="heading with text/comment/text nodes">
+  heading<!-- with non-text node splitting concatenated text nodes -->label<!-- [sic] no extra spaces around first comment -->
+</h2>
```

Comment:
do you know why this fails on chrome without the useful information...? the other failures showed why the comparison failed.

Reply:
Click on the "show details" switch (above the table on the right side) on that [results page](https://wpt.fyi/results/accname/name/comp_text_node.html?diff&filter=ADC&run_id=6197724825780224&run_id=5169589548220416):

```
FAIL message: assert_equals: <h2 class="ex" data-expectedlabel="heading label" data-testname="heading with text/comment/text nodes" id="labelTest1">
  heading<!-- with non-text node splitting concatenated text nodes -->label<!-- [sic] no extra spaces around first comment -->
</h2> expected "heading label" but got "headinglabel"
```

That looks like a valid failure to me, relatively inconsequential and with an easy author workaround.

Reply:
Even though I think WebKit's behavior is to spec here, there is some merit in the Chromium implementation. I will file another issue and skip this test until it's resolved.

Reply:
https://github.com/w3c/accname/issues/193

Reply:
https://github.com/web-platform-tests/wpt/pull/39604/commits/f0421a91bc38df7bc29928da2d17aee0d4ac5cd2